runtime/j9vm/jvm.c

```diff
@@ -4073,9 +4119,13 @@ JVM_FindLibraryEntry(void* handle, const char *functionName)
 
 #if defined(WIN32)
 	result = GetProcAddress ((HINSTANCE)handle, (LPCSTR)functionName);
-#elif defined(J9UNIX) || defined(J9ZOS390) /* defined(WIN32) */
-	result = (void*)dlsym( (void*)handle, (char *)functionName );
-#else /* defined(WIN32) */
+#elif defined(J9ZOS390) /* defined(WIN32) */
+	PORT_ACCESS_FROM_JAVAVM(BFUjavaVM);
+	/* Call j9sl_lookup_name to handle potential 31-bit interoperability targets. */
+	j9sl_lookup_name((UDATA)handle, (char *)functionName, (void *)&result, "");
+#elif defined(J9UNIX) /* defined(J9ZOS390) */
+	result = (void *)dlsym((void *)handle, (char *)functionName );
```

Comment:
Since you're changing this line, please remove the space before the closing `)`.

Reply:
Fixed in 55ae9d3